mosaic-site/src/css/global/code.css

```diff
@@ -0,0 +1,158 @@
+/**
+ * prism.js default theme for JavaScript, CSS and HTML
+ * Based on dabblet (http://dabblet.com)
+ * @author Lea Verou
+ */
+
+pre {
+  --pre-background-color: #f6f8fa;
+  --pre-border-radius: 0.4rem;
+  --pre-color: #393a34;
+  --code-font-size: 90%;
+  --pre-line-height: 1.45;
+  --font-family-monospace: SFMono-Regular, Menlo;
+  --pre-shadow: 0 1px 2px 0 #0000001a;
+}
```

Comment:
may want to look and redefine these in future

Reply:
Agreed! As I mentioned above we're just trying to match our current styling for now but we should definitely revisit